packages/response-ops/alerts_table/apis/bulk_get_cases.test.ts

```diff
@@ -1,14 +1,16 @@
 /*
  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
 import { bulkGetCases } from './bulk_get_cases';
 import { coreMock } from '@kbn/core/public/mocks';
 
-describe('Bulk Get Cases API', () => {
+describe(bulkGetCases, () => {
```

Comment:
Trying out this new way to name describe blocks using the tested symbol directly. This should avoid us a lot of misaligned names in case of refactors but has the downside of not working with memoized/forwardRef'ed React components because they are not normal functions.